### PR TITLE
Menu item and interface for data download

### PR DESCRIPTION
**Pull request by marekviktor**
### Contributor checklist

- [] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Related issue

- #ISSUE_NUMBER

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks, @marekviktor! I'll check this in the coming days :)

**Comment by andrewtavis** [comment]
Hey @marekviktor 👋 Just a quick note on this: we checked this in the dev sync today and it's looking great 😊 I'll do a more  full review on Monday or Tuesday as I'm flying tomorrow ✈️

**Comment by andrewtavis** [comment]
There are some changes that are needed, but I was thinking we could do some minor issues for those. You'd be very welcome to pick those up!

**Comment by SaurabhJamadagni** [comment]
Hey @andrewtavis could you please make sure I haven't got rid of some of the recent changes during the rebase. Scribe is on a roll with the contributions!! There were just too many commits to rebase from. 

Currently I have commented the two important files (the DownloadData and DataDownloadVC files) that @marekviktor has contributed as they are referencing elements not there in the current setup. We can later make the right provisions to include them. I hope that works.

**Comment by andrewtavis** [comment]
Sounds perfect, @SaurabhJamadagni! Appreciate you putting in all this effort :) I'll try to look into all of this soon, or maybe we could also check in the sync on Saturday. 

@marekviktor, sorry for the wait on this! There's been lots going on recently, but we'll try to get this in soon! Again, really appreciate the contribution :)

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -17,23 +17,23 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for our
```
Comment (wkyoshida) [inline .github/CODE_OF_CONDUCT.md:17]: Looking here - I'm thinking that everything here inside:
- `.github/`
- `README.md`
- `Scribe-i18n/`

can get removed probably. Looks like they really are from consolidating previous commits. 

Looks like the things that we'd want are what is under:
- `Scribe.xcodeproj/`
- `Scribe/`

as we can kinda see in the original commit for the PR, 80aa484d52f945335594d7589b87da07fd02839e

**Pull request merged**